esp/src/src-react/components/Files.tsx

```diff
@@ -116,6 +117,16 @@ export const Files: React.FunctionComponent<FilesProps> = ({
         total, setTotal,
         refreshTable } = useFluentStoreState({ page });
 
+    //  Refresh on focus  ---
+    const isMount = useIsMount();
+    const hasFocus = useFocusChange();
+    React.useEffect(() => {
+        if (hasFocus && !isMount) {
```

Comment:
Mount and Unmount are React terms for when the Component enters and exits the page.

We don't want to refresh the grid during the initial Mount as it would cause the grid to fetch the data twice.

IOW `isMount` will be true only for the first iteration of this function.

Reply:
ok.  The definitions of isMount I found seemed like they were more to do with whether the object was fully constructed/destroyed.. which didn't make much sense.

e.g. https://usehooks-ts.com/react-hook/use-is-mounted

I don't pretend to understand it...

Reply:
Comparing to the code in the link. my hook is more an "isMounting" (and it doesn't care about unmounting). 

I might change to use your example, as it would be more reusable in the future...

Reply:
The usage here (and in Queries.tsx) didn't change based on the latest changes to the hook, though Workunits did.